### PR TITLE
Post 1.26.0 release updates

### PR DESCRIPTION
**Pull request by ulyssessouza**

### CODE REVIEW

File: compose/release-notes.md [hunk 1]
```diff
@@ -7,6 +7,45 @@ redirect_from:
...
+
+- Synchronized compose-schemas with upstream (docker/cli)
+
+- Removed `None` entries on exec command
+
+- Added `distro` package to get distro information
+
+- Added `python-dotenv` to delegate `.env` file processing
+
+- Stoped adjusting output on terminal width when piped into another command
```
Comment (stephen-turner) [inline compose/release-notes.md:43]: Stopped

Comment (ulyssessouza) [reply compose/release-notes.md:43]: Ops... Thanks

File: compose/release-notes.md [hunk 2]
```diff
@@ -7,6 +7,45 @@ redirect_from:
...
+
+- Bumped `OpenSSL` from 1.1.1f to 1.1.1g
+
+- Bumped `certifi` from 2019.11.28 to 2020.4.5.1
+
+- Bumped `docker-py` from 4.2.0 to 4.2.1
+
+### Bug fixes
+
+- Properly escapeed values coming from env_files
```
Comment (usha-mandya) [inline compose/release-notes.md:33]: escaped

Comment (ulyssessouza) [reply compose/release-notes.md]: fixed

**Pull request merged**